packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -0,0 +1,223 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  Widget buildTest(
+    Key box1Key,
+    Key box2Key,
+    Key box3Key,
+    ScrollController controller, {
+    Axis? axis,
+  }) {
+    return Directionality(
+      textDirection: TextDirection.ltr,
+      child: ScrollConfiguration(
+        behavior: const ScrollBehavior().copyWith(overscroll: false),
```

Comment:
should this just pass in a behavior that has the new stretch enabled instead of deactivating it and then manually wrapping?

Reply:
I was going to write separate tests for the scroll behavior once I settled on an opt-in. I had a think over the weekend and I think I have a better idea.